administration/src/application/ApplicationErrorBoundary.tsx

```diff
@@ -0,0 +1,35 @@
+import localforage from 'localforage'
+import { Component, ReactNode } from 'react'
+import { applicationStorageKey } from './components/ApplyController'
+import { globalArrayBuffersKey } from './globalArrayBuffersManager'
+
+class ApplicationErrorBoundary extends Component<{ children: ReactNode }, { resetting: boolean }> {
+  state = { resetting: false }
+
+  async componentDidCatch() {
+    this.setState({ resetting: true })
+    try {
+      await localforage.removeItem(applicationStorageKey)
+      await localforage.removeItem(globalArrayBuffersKey)
+    } catch (e) {
+      console.error('Another error occurred while resetting the application state.', e)
+    } finally {
+      this.setState({ resetting: false })
+    }
+  }
+
+  render() {
+    if (this.state.resetting) return null
```

Comment:
Add braces. Shouldn't such things be cought by our linter?

Reply:
probably our linter allows that currently.